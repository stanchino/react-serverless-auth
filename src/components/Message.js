import React from "react";

export default ( { message, style }) => (
    message ? <div style={style}><strong>{message}</strong></div> : null
);