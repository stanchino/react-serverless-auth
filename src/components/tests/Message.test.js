import { matchSnapshot } from "./shared-examples";

import Message from "../Message";

describe("Message", () => {
    it("matches the snapshot", () => matchSnapshot(Message));
});