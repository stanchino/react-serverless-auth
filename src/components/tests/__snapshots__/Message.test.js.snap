// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message matches the snapshot 1`] = `null`;
